recipes/baycomp/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "baycomp" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c43472c16bd7cdf4884dd4c73dd307e4a3da7097fe49c83cd5e88d75142923b0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python
```

You will need to limit ``python`` here as well, could you please add it?
```suggestion
    - python >=3.6
```